### ISSUE TITLE
Question about data

### ISSUE THREAD

**Issue by Jingyilang** [xueyouluo/NER-Deep-Learning#1]
Hi, thranks for your code first. Could you tell where is the "example" data set from? is it People's Daily98?

**Comment by xueyouluo** [comment]
No, it is not People 98. I copied from this project https://github.com/zjy-ucas/ChineseNER. I think it maybe bakeoff2006 data.  You can use People's Daily data and Boson NLP data, just convert them to the format as the example data.
